server/src/main/java/org/cloudfoundry/identity/uaa/audit/JdbcFailedLoginCountingAuditService.java

```diff
@@ -52,6 +52,16 @@ public void log(AuditEvent auditEvent) {
                                                 - saveDataPeriodMillis));
                 super.log(auditEvent);
                 break;
+            case ClientAuthenticationSuccess:
```

Comment:
Okay, will do. Should the revisions be in a separate commit or should I tack it onto the existing commit?